spec/order_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "order"
+
+describe Order do
+  subject(:order) { described_class.new }
+  let(:menu) { double(:menu) }
```

Comment:
You've made a double, but you didn't do anything with it. I think you can skip this line!